components/Record/Hashtags.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {
+  View,
+  StyleSheet,
+  TextInput,
+  KeyboardAvoidingView,
+} from "react-native";
+
+interface PropTypes {
+  hashtagSetter: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
Tar vi hand om att ta bort dubbletter av hashtags samt den tomma hashtagen någonstans än? Annars kanske du kan slänga in en TODO på det :D

Reply:
Kan slänga in den i scopet på [ticketen](https://app.asana.com/0/1201301746458974/1201431345772993) att cleana up den här 👷